templates/base.html

```diff
@@ -0,0 +1,99 @@
+<!doctype html>
+<html lang="en">    
+<!-- This html provides the base header and footer for all html files except upload.html -->
+<head>
+      <title>{% block title %}{% endblock %} - Flaskr</title>
+      <link rel="stylesheet" href="{{ url_for('static', filename='style.css') }}">
+      <meta charset="utf-8">
+      <meta http-equiv="X-UA-Compatible" content="IE=edge">
+      <meta name="viewport" content="width=device-width, initial-scale=1">
+      <!-- Link to stylesheet for upload form-->
+      <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css" integrity="sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO" crossorigin="anonymous">
+      <!-- Link to stylesheet main.css, not important for this webpage-->
+      <!--link rel="stylesheet" href="Assets/main.css"--> 
+      <!-- Link to stylesheet for the font and the horizontal bar on the top-->
+      <link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
+  </head>
+
+<section class="content">
+  
+  <header>
+    <body>
+      <div class="w3-container w3-dark-grey">
+
+  <p></p>
+</div>
+        <!-- Site header and link to other parts of the website. Links will need to be updated if website domain changes. -->
+        <header class="site-header" role="banner">
+            <ka>
+              <div class="wrapper" style="margin-left: 25%;"><a class="site-title" rel="author" href="/"><font size="6">West Oakland Open Air Quality Data</font></a>&emsp;&emsp;&emsp;<font size="4"> <a href="https://openoakland.github.io/woaq/about/">About</a> </font>&emsp;&emsp;&emsp;<font size="4"> <a href="/upload">Upload</a> </font><nav class="site-nav"> 
+              </div>
+              </nav></div>
+          </ka>
+            <hr size=2 width=100%>
+        </header>
+    {% block header %}{% endblock %}
+  </header>
+  {% for message in get_flashed_messages() %}
+    <div class="flash">{{ message }}</div>
+  {% endfor %}
+  {% block content %}{% endblock %}
+
+          </kd>
+        <br>
+        <br>
+        <br>
+        <br>
+        <br>
+        <br>
+        <br>
+        <br>
+        <br>
+        <br>
+
+        <!-- Area for Further Resource links. Links do need to be updated unless outside webpages change. -->
+        <hr size=2 width=100%>
+        <kd>
+        <footer class="site-footer h-card">
+          <font size="3">
+          <data class="u-url" href="/woaq/"></data>
+
+          <div class="wrapper" style="margin-left:25%;margin-right:25%;">
+
+
+            <h2 class="footer-heading">Further Resources</h2>
+
+            <div class="footer-col-wrapper">
+              <div class="footer-col footer-col-1">
+                <ul class="contact-list">
+                  
+                  <li class="p-name">
+                    <a href="https://www.youtube.com/watch?time_continue=143&amp;v=KThvnkzSatU">WOEIP Video</a>
```

Comment:
To improve accessibility for people using screen readers, we should add properties such as a label